components/MapDisplay.js

```diff
@@ -0,0 +1,69 @@
+class MapDisplay {
+  constructor(app) {
+    this.app = app;
+    this.latitudeElement = document.querySelector(".latitude-text");
+    this.longitudeElement = document.querySelector(".longitude-text");
+    this.loadingText = document.querySelector(".loading-text");
+
+    this.map = null;
+    this.placemark = null;
+  }
+
+  successCallback(position) {
+    const latitude = position.coords.latitude;
+    const longitude = position.coords.longitude;
+
+    this.latitudeElement.textContent = `Latitude: ${latitude
+      .toString()
+      .slice(0, 5)
+      .replace(".", "° ")}'`;
+    this.longitudeElement.textContent = `Longitude: ${longitude
+      .toString()
+      .slice(0, 5)
+      .replace(".", "° ")}'`;
```

Comment:
Same to temperature, text should be treated separately from value

Reply:
https://github.com/VasiliiDubik/Vasiliidubik-weatherForecast/pull/1/commits/08771654591007da06e84007d87335d2ef8f5ebf
The change concerns longitude and latitude, not temperature; this approach ensures that the information is displayed according to the Figma design. I also tried to improve the code readability by creating a separate function called formatCoordinates.